lbz/exceptions.py

```diff
@@ -18,10 +18,14 @@ class LambdaFWException(Exception):
     message = HTTPStatus.INTERNAL_SERVER_ERROR.description
     status_code = HTTPStatus.INTERNAL_SERVER_ERROR.value
 
-    def __init__(self, message: str = ""):
+    def __init__(self, message: str = None, log_msg: str = None, log_warn: str = None):
```

Comment:
I know it was moved from the `ServerError` but it is not the best way how messages should be logged.

A couple of reasons of why:
1. Each module should contain its own logger instance, just to have an easy way to get knowledge of where the log comes from. By doing that this way, all error/warning logs will be marked as from the exceptions module.
2. In most cases, the error logs should be registered on Exceptions handling, not on raising them. Of course, it depends on the situation in which the error appears but it is very possible that this feature won't be used that often.
3. The messages to the logger shouldn't be passed as a formatted string (rather as a message and its arguments) - by doing that this way, we lose that possibility.
4. The line with logger is that short that it shouldn't be a problem to write it in a specific place. From my perspective, this feature is not that beneficial (does not save duplicating a lot of code).

Reply:
That is a good moment to extend the `__init__` with the arguments for the base class 😉 
```python
def __init__(self, message: str = None, *args, **kwargs):  # type: ignore
    super().__init__(*args, **kwargs)  # type: ignore
```

Reply:
args are fine Exception._iinit__ does not take **kwargs ;) but otherway around adjusted.

Reply:
That's right - only PyCharm's stubs are saying something else. Anyway, I wanted to stay compatible with builtins 😉